examples/data-objects/multiview/constellation-model/src/model.ts

```diff
@@ -57,12 +57,14 @@ export class Constellation extends DataObject implements IConstellation {
     }
 
     private async updateStarsFromRoot() {
-        const starHandles = this.root.get<IFluidHandle<ICoordinate>[]>(starListKey);
+        // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+        const starHandles = this.root.get<IFluidHandle<ICoordinate>[]>(starListKey)!;
```

Comment:
Naive question: is this safe? Do we have some guarantees in the framework that this won't be null?

Reply:
nope. non-null is a type assertion (it remove both undefined and null from the type)  It is same as `as`, which doesn't have any runtime code.